src/main/java/domain/LottoStatisticData.java

```diff
@@ -0,0 +1,81 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class LottoStatisticData {
+    private final LottoNumbers winnerNumbers;
+    private final LottoNumber bonusNumber;
+    private final HashMap<Match, Integer> matchStatistic;
```

Comment:
인터페이스인 `Map`으로 선언하는게 더 좋아보여요!
```suggestion
    private final Map<Match, Integer> matchStatistic;
```